src/components/basic/Datepicker/index.tsx

```diff
@@ -93,6 +91,48 @@ export const Datepicker = ({
   }
 
   const iconColor = open ? daySelectedColor : '#939393'
+
+  const ServerDay = (props: any) => {
+    const { selected, day, today } = props
```

Comment:
we are using these three attributes from props so let's define and use an interface instead of any. Because in the current sprint the any's will be removed anyway.